docs/index.qmd

```diff
@@ -40,58 +41,317 @@ about:
+
+```{python}
+#| code-fold: true
+#| echo: false
+
+import ibis
+
+t = ibis.examples.penguins.fetch()
+t.to_parquet("penguins.parquet")
 ```
 
-1. Install Ibis with the DuckDB backend along with examples.
+## Fast local dataframes
 
-<div class="d-grid gap-2"><a class="btn btn-lg btn-primary" data-bs-toggle="collapse" href="#collapseBackends" role="button" aria-expanded="false" aria-controls="collapseBackends"  margin="100px">Show supported backends</a></div>
+Ibis offers a familiar local dataframe experience with outstanding performance,
+using [DuckDB](https://duckdb.org) by default.
 
-###
+```{python}
+import ibis  # <1>
 
-::: {#collapseBackends .collapse .multi-collapse}
+ibis.options.interactive = True  # <2>
 
-## Backends
+t = ibis.read_parquet("penguins.parquet", table_name="penguins")  # <3>
+t.head(3)  # <4>
+```
 
-Need to use Ibis with a backend that isn't currently supported? [Let us know!](https://github.com/ibis-project/ibis/discussions/new?category=q-a)
+1. Import Ibis.
+2. Enable interactive mode for exploratory data analysis (EDA) or demos.
+3. Read a Parquet file and specify a table name (optional).
+4. Display the first few rows of the table.
 
-{{< include ./_tabsets/install.qmd >}}
+Iterate and explore data locally:
 
-See the [backend support matrix](support_matrix.qmd) for details on operations supported. [Open a feature request](https://github.com/ibis-project/ibis/issues/new?assignees=&labels=feature&projects=&template=feature-request.yml&title=feat) if you'd like to see support for an operation in a given backend. If the backend supports it, we'll do our best to add it quickly!
+```{python}
+grouped = t.group_by("species", "island").agg(count=t.count()).order_by("count")  # <1>
+grouped  # <2>
+```
+
+1. Transform the table.
+2. Display the transformed table.
+
+## Python + SQL: better together
+
+Ibis works by decoupling the dataframe API from the backend execution. Most
+backends support a SQL dialect, which Ibis compiles its expressions into using
+[SQLGlot](https://github.com/tobymao/sqlglot). You can inspect the SQL that Ibis
+generates for any SQL backend:
+
+```{python}
+ibis.to_sql(grouped)  # <1>
+```
+
+1. Display the SQL generated from the table expression.
+
+And use SQL strings directly, mixing and matching with Python dataframe code:
+
+```{python}
+t.sql(  # <1>
+    "SELECT species, island, COUNT(*) AS count FROM penguins GROUP BY species, island"  # <1>
+).order_by("count")  # <2>
+```
+
+1. Transform the table using SQL.
+2. Then, transform the table using Python dataframe code.
+
+This allows you to combine the flexibility of Python with the scale and
+performance of modern SQL.
+
+## 20+ backends
+
+Use the same dataframe API for 20+ backends:
+
+```{python}
+#| code-fold: true
+#| echo: false
+
+from backends_sankey import fig
+fig.show()
+```
+
+For example:
+
+::: {.panel-tabset}
+
+## DuckDB
+
+```{python}
+backend = "duckdb"
+con = ibis.connect(f"{backend}://")
```

Comment:
I don't particularly like the use of `backend` variable + f-string here; why have all the tabs and then use substitution, think you can pick one.